WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/ShippingLabelPaperSizeOptionsView.swift

```diff
@@ -0,0 +1,51 @@
+import Networking
+import SwiftUI
+
+/// Displays a grid view of all available paper size options for printing a shipping label.
+struct ShippingLabelPaperSizeOptionsView: View {
+    private let paperSizeOptions: [ShippingLabelPaperSize]
+    private let numberOfColumnsPerRow = 2
+    private let numberOfRows: Int
+
+    init(paperSizeOptions: [ShippingLabelPaperSize]) {
+        self.paperSizeOptions = paperSizeOptions
+        self.numberOfRows = Int(ceil(Double(paperSizeOptions.count) * 1.0 / Double(numberOfColumnsPerRow)))
```

Comment:
Maybe this can be extracted to a function? Also, why are we multiplying by 1?

Reply:
Ah it was a habit to multiple by 1.0 to make sure it's a floating number, but it isn't necessary here since the `Int` values are already cast to `Double`. Removed in: https://github.com/woocommerce/woocommerce-ios/pull/3322/commits/520d99d926516915236b10b0429c77b8de632fd6

> Maybe this can be extracted to a function?

it's a bit tricky to call a function in the initializer, and it's mostly casting / handling two numbers. but lemme know if you think any way of writing would make it easier to read!